src/status_im/contexts/wallet/send/select_address/view.cljs

```diff
@@ -156,16 +155,13 @@
                                                          local-suggestion-address
                                                          input-value)
                                    [_ splitted-address] (network-utils/split-network-full-address
-                                                         address)]
-                               (rf/dispatch
-                                [:wallet/select-send-address
-                                 {:address address
-                                  :recipient {:label
-                                              (utils/get-shortened-address
-                                               splitted-address)
-                                              :recipient-type :address}
-                                  :stack-id
-                                  :screen/wallet.select-address}])))
+                                                         address)
+                                   recipient            {:label
+                                                         (utils/get-shortened-address
+                                                          splitted-address)
+                                                         :recipient-type :address}]
```

Comment:
The code style here looks very uncommon. consider extracting a call to another binding so that this map is formatted consistently:

Either:
```clojure
{:a 'a
 :b 'b}
```
or (ugly):
```clojure
{:a
 'a
 :b
 'b}
```
But mixing these is not great